docs/styles/classes.md

```diff
@@ -61,6 +61,21 @@ non-default `bodyOpenClassName`), you could use the following CSS:
 }
 ```
 
+You can define a class to be added to the html tag, using the `htmlOpenClassName`
+attribute, which can be helpeful to stop the page to scroll to the top when open
+a modal.
```

Comment:
@woniesong92 Ah...I think I could add something here to remind that it's `null`.